appium/tests/screenobjects/keys.screen.ts

```diff
@@ -1,12 +1,13 @@
 import BaseScreen from './base.screen';
 import ElementHelper from "../helpers/ElementHelper";
+import {ekmPrivateKeySamples, KeyDetailInfo} from "../../api-mocks/apis/ekm/ekm-endpoints";
 
 const SELECTORS = {
   KEYS_HEADER: '-ios class chain:**/XCUIElementTypeStaticText[`label == "Keys"`]',
   ADD_BUTTON: '~Add',
-  NAME_AND_EMAIL: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[1]',
-  DATE_CREATED: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[2]',
-  FINGERPRINT: '-ios class chain:**/XCUIElementTypeOther/XCUIElementTypeStaticText[3]',
+  NAME_AND_EMAIL: '~keyTitle',
+  DATE_CREATED: '~keyDate',
+  FINGERPRINT: '~keySubtitle',
   SHOW_PUBLIC_KEY_BUTTON: '~Show public key',
   SHOW_KEY_DETAILS_BUTTON: '~Show key details',
   COPY_TO_CLIPBOARD_BUTTON: '~Copy to clipboard',
```

Comment:
These should all ideally use custom `aid-`. I'll file another issue for that for later.

Reply:
#1545